dvc/repo/gc.py

```diff
@@ -82,7 +82,7 @@ def gc(
         return
 
     odb = self.cloud.get_remote_odb(remote, "gc -c")
-    removed = ogc(odb, used_obj_ids)
+    removed = ogc(odb, used_obj_ids, jobs=jobs)
```

Comment:
We have an issue for that: https://github.com/iterative/dvc/issues/5961

Reply:
I'm looking into how easy it would be to add basic multithreading to `gc`. For now, it only uses multithreading when listing objects from the odb (`odb.all()`) using `_list_hashes_traverse`.

I think it should #5961 should be closed in another PR.